packages/shift-next/src/actions/account-actions.js

```diff
@@ -18,6 +18,37 @@ export function fetchAccountDetails (options) {
   return readEndpoint(accountRequest, options)
 }
 
+export function updateCustomerPassword (details, options) {
```

Comment:
I know this is a name given from the previous handler, just wondered if we can rename this more appropriately?

```suggestion
export function updateCustomerPassword (formData, options) {
```